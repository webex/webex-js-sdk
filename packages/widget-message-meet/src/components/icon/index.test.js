import React from 'react';
import renderer from 'react-test-renderer';

import Icon from '.';

describe(`Icon component`, () => {

  const component = renderer.create(
    <Icon
      type="delete"
    />
  );

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
