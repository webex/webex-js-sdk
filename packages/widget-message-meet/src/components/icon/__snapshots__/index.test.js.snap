exports[`Icon component renders properly 1`] = `
<div
  className="icon"
  title={undefined} />
`;
